core/src/main/java/com/linecorp/armeria/server/ServerErrorHandler.java

```diff
@@ -236,4 +237,26 @@ public AggregatedHttpResponse renderStatus(ServiceConfig config,
             }
         };
     }
+
+    /**
+     * Casts this {@link ServerErrorHandler} to {@link ServiceErrorHandler}.
+     * @return {@link ServiceErrorHandler} cast from this {@link ServerErrorHandler}.
```

Comment:
We're not casting, the `@return` description doesn't really convey additional information

```suggestion
     * Transforms this {@link ServerErrorHandler} into a {@link ServiceErrorHandler}.
```